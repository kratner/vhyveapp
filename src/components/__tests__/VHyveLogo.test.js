import React from "react";
import { shallow } from "enzyme";
//import { mount } from "enzyme";
import VHyveLogo from "../VHyveLogo";

describe("VHyveLogo", () => {
  it("renders correctly", () => {
    /*
    const props = {
      "data-ctl": "bgvideoswitch",
      className: "ctl ctl__bgvideoswitch",
      title: "Switch Video Background"
    };
    */
    // const wrapper = shallow(<VHyveLogo {...props} />);
    const wrapper = shallow(<VHyveLogo delay="2000" />);
    expect(wrapper).toMatchSnapshot();
  });
});
