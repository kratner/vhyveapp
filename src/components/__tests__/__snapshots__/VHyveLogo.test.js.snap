// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VHyveLogo renders correctly 1`] = `ShallowWrapper {}`;
